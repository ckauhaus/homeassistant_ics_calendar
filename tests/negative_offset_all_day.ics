BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//team_2_nl//NONSGML kigkonsult.se iCalcreator 2.20.2//
X-WR-TIMEZONE:Europe/Brussels
BEGIN:VEVENT
UID:match_1025179
DTSTAMP:20210911T014015Z
DESCRIPTION:Wedstrijd van Speeldag 1 uit de Europa League  tussen Olympiako
 s Piraeus en Antwerp.\n\nOlympiakos Piraeus wint @ 1.76\nGelijk @ 4.00\nAn
 twerp wint @ 4.35\nWed op https://banners.livepartners.com/click.php?id=34
 9374&p=10&t=betslip&MasterEventID=24267084\n\nOnderlinge duels:\n\n09/12/2
 021 Antwerp - Olympiakos Piraeus: 0-0\n16/09/2021 Olympiakos Piraeus - Ant
 werp: 0-0
DTSTART:20210916
DTEND:20210917
SUMMARY:Olympiakos Piraeus - Antwerp
END:VEVENT
END:VCALENDAR
