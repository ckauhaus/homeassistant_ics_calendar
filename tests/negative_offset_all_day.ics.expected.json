[
    {
        "start": "2021-09-16",
        "end": "2021-09-17"
    }
]
